packages/adblocker-benchmarks/Makefile

```diff
@@ -56,6 +64,9 @@ ublock:
 adblockplus:
 	$(node_command) run.js adblockplus $(run_options)
 
+tsurlfilter:
```

Comment:
Nit. Can we maybe call this `adguard` to make it clearer?

Reply:
>Can we maybe call this `adguard` to make it clearer?

It's not used in the AdGuard browser extension yet. I thought we should just call it by the name of the library for now. When it's integrated into the AdGuard browser extension, we can change the name here.

Reply:
>It's not used in the AdGuard browser extension yet. I thought we should just call it by the name of the library for now. When it's integrated into the AdGuard browser extension, we can change the name here.

@remusao I'll make this as resolved but let me know if you disagree.